packages/admin/dashboard/src/routes/promotions/promotion-edit-details/components/edit-promotion-form/edit-promotion-details-form.tsx

```diff
@@ -247,8 +250,8 @@ export const EditPromotionDetailsForm = ({
                           onValueChange={(val) =>
                             onChange(val ? parseInt(val) : null)
                           }
-                          code={"USD"}
-                          symbol={getCurrencySymbol("USD")}
+                          code={currencyCode}
```

Comment:
this was hardcoded all this time? 😄

Reply:
Yup :D it seems only here though, I did some quick search